mlibrary/catalog-browse
# [LIBSEARCH-856] Update links from Subject Browse to Catalog Search to use "LC Subject starts with" fielded search

# Overview
With `LC Subject starts with` option being added, the subject browse links will now be linked to that query.

## Testing
- Run the tests to make sure they pass (`docker-compose run --rm web bundle exec rspec`).
  - Break the new/updated unit tests to make sure they're working properly.
- Make sure the PR is consistent in these browsers:
- Run accessibility tests:
- Check to see if the new links will lead you to a `subject_browse_leftanchored` queried search.
